bndtools.core/bndtools.gtk.linux-eclipse-neon..x86.bndrun

```diff
@@ -0,0 +1,18 @@
+# use this runfile when you are on GTK 3.20+ 
+# because the version from Eclipse 4.4.2 doesnt work with newer versions of GTK
+
+-include: bndtools.shared.bndrun
+
+-runbundles: \
+	${crossplatform.bundles},\
+	org.eclipse.core.filesystem.linux.x86,\
+	org.eclipse.core.net.linux.x86,\
+	org.eclipse.swt.gtk.linux.x86
```

Comment:
How about here for the swt bundle we add the version base that we know will have to work, in case we ever update the version it the build repo, adding this below will ensure at least that version is found.  
```
org.eclipse.swt.gtk.linux.x86;version='3.105.2'
```